pages/bad-page.js

```diff
@@ -12,49 +11,45 @@ const betterButtons = [
 ];
 
 export default function BadPage() {
-  const router = useRouter();
-  const [buttons, setButtons] = useState(betterButtons);
+  const [aspects, setAspects] = useState(initialAspects);
+  const activeAspects = aspects.filter((aspect) => aspect.status).length;
 
-  function handleSubmit() {
-    window.localStorage.setItems(`buttons`, JSON.stringify(buttons));
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    console.log(data);
   }
```

Comment:
since you want to navigate to another page after submitting your form, you can work with `router.push()` in your `handleSubmit()`

```suggestion
  function handleSubmit(event) {
    event.preventDefault();
    const formData = new FormData(event.target);
    const data = Object.fromEntries(formData);
   
   router.push('/thanks')
  }
```